Define ScyllaDB as drop-in replacement for Cassandra (#4353)
jaegertracing/jaeger: README.md

* It natively supports two popular open source NoSQL databases as trace storage backends: Cassandra and Elasticsearch.
* It integrates via a gRPC API with other well known databases that have been certified to be Jaeger compliant: [TimescaleDB via Promscale](https://github.com/timescale/promscale), [ClickHouse](https://github.com/jaegertracing/jaeger-clickhouse).
* There is embedded database support using [Badger](https://github.com/dgraph-io/badger) and simple in-memory storage for testing setups.

> * It supports ScyllaDB as a drop-in replacement for Cassandra because ScyllaDB uses the same data model and query language as Cassandra.

Reviewer: ```suggestion
* ScyllaDB can be used as a drop-in replacement for Cassandra since it uses the same data model and query language.
```
